src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
```

Comment:
The getSum function should take two arguments according to the task description but it only takes one in your implementation. Please adjust this.

Reply:
fixed